script.js

```diff
@@ -0,0 +1,90 @@
+window.onload = start;
+var boxes = document.getElementsByTagName("td");
+var turnText = document.querySelector(".playerTurn");
+var counter = 1;
```

Comment:
In the future, I might try to eliminate these global variables, but it certainly still works fine the way you currently have it set up